2024-08-15 11:17:10,489 INFO    MainThread:24029 [wandb_setup.py:_flush():76] Current SDK version is 0.16.4
2024-08-15 11:17:10,489 INFO    MainThread:24029 [wandb_setup.py:_flush():76] Configure stats pid to 24029
2024-08-15 11:17:10,489 INFO    MainThread:24029 [wandb_setup.py:_flush():76] Loading settings from /root/.config/wandb/settings
2024-08-15 11:17:10,489 INFO    MainThread:24029 [wandb_setup.py:_flush():76] Loading settings from /mnt/c/Users/jahaz/OneDrive/Escritorio/Test/Machine-Learning-DevOps-Engineer/03_Building_reproducible_model_workflow/03_data_exploration_and_preparation/03_data_segregation/solution/wandb/settings
2024-08-15 11:17:10,489 INFO    MainThread:24029 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-08-15 11:17:10,490 INFO    MainThread:24029 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-08-15 11:17:10,491 INFO    MainThread:24029 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': '03_Building_reproducible_model_workflow/03_data_exploration_and_preparation/03_data_segregation/solution/run.py', 'program_abspath': '/mnt/c/Users/jahaz/OneDrive/Escritorio/Test/Machine-Learning-DevOps-Engineer/03_Building_reproducible_model_workflow/03_data_exploration_and_preparation/03_data_segregation/solution/run.py', 'program': '/mnt/c/Users/jahaz/OneDrive/Escritorio/Test/Machine-Learning-DevOps-Engineer/03_Building_reproducible_model_workflow/03_data_exploration_and_preparation/03_data_segregation/solution/run.py'}
2024-08-15 11:17:10,491 INFO    MainThread:24029 [wandb_init.py:_log_setup():526] Logging user logs to /mnt/c/Users/jahaz/OneDrive/Escritorio/Test/Machine-Learning-DevOps-Engineer/03_Building_reproducible_model_workflow/03_data_exploration_and_preparation/03_data_segregation/solution/wandb/run-20240815_111710-kuhhfczb/logs/debug.log
2024-08-15 11:17:10,492 INFO    MainThread:24029 [wandb_init.py:_log_setup():527] Logging internal logs to /mnt/c/Users/jahaz/OneDrive/Escritorio/Test/Machine-Learning-DevOps-Engineer/03_Building_reproducible_model_workflow/03_data_exploration_and_preparation/03_data_segregation/solution/wandb/run-20240815_111710-kuhhfczb/logs/debug-internal.log
2024-08-15 11:17:10,492 INFO    MainThread:24029 [wandb_init.py:init():566] calling init triggers
2024-08-15 11:17:10,492 INFO    MainThread:24029 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {}
2024-08-15 11:17:10,492 INFO    MainThread:24029 [wandb_init.py:init():616] starting backend
2024-08-15 11:17:10,492 INFO    MainThread:24029 [wandb_init.py:init():620] setting up manager
2024-08-15 11:17:10,496 INFO    MainThread:24029 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-08-15 11:17:10,497 INFO    MainThread:24029 [wandb_init.py:init():628] backend started and connected
2024-08-15 11:17:10,504 INFO    MainThread:24029 [wandb_init.py:init():720] updated telemetry
2024-08-15 11:17:10,576 INFO    MainThread:24029 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-08-15 11:17:11,022 INFO    MainThread:24029 [wandb_run.py:_on_init():2262] communicating current version
2024-08-15 11:17:11,116 INFO    MainThread:24029 [wandb_run.py:_on_init():2271] got version response upgrade_message: "wandb version 0.17.6 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-08-15 11:17:11,117 INFO    MainThread:24029 [wandb_init.py:init():804] starting run threads in backend
2024-08-15 11:17:11,973 INFO    MainThread:24029 [wandb_run.py:_console_start():2241] atexit reg
2024-08-15 11:17:11,974 INFO    MainThread:24029 [wandb_run.py:_redirect():2096] redirect: wrap_raw
2024-08-15 11:17:11,975 INFO    MainThread:24029 [wandb_run.py:_redirect():2161] Wrapping output streams.
2024-08-15 11:17:11,975 INFO    MainThread:24029 [wandb_run.py:_redirect():2186] Redirects installed.
2024-08-15 11:17:11,976 INFO    MainThread:24029 [wandb_init.py:init():847] run started, returning control to user process
2024-08-15 11:17:27,078 WARNING MsgRouterThr:24029 [router.py:message_loop():77] message_loop has been closed
